8.1: What's new in Observability? (#1643)
elastic/observability-docs: docs/en/observability/whats-new.asciidoc

Here are the highlights of what's new and improved in {minor-version}.

Other versions:
{observability-guide-all}/8.1/whats-new.html[8.1] |
{observability-guide-all}/8.0/whats-new.html[8.0] |
{observability-guide-all}/7.17/whats-new.html[7.17] |

// tag::whats-new[]
[discrete]
== Fleet now works with limited privileges

Fleet and Elastic Agent Integrations previously required a superuser role.
This required organizations to give users very broad access,
which created more potential for security and operational risks.

> You can now grant users a more limited set of Kibana privileges for Fleet and Integrations.
> This will allow you to safely grant access to more users within your organization.
> No Elasticsearch privileges are required since Kibana handles updating Elasticsearch assets automatically.

Author: ```suggestion
Starting in 8.1, you can grant users a more limited set of Kibana privileges for Fleet and Integrations.
No Elasticsearch privileges are required since Kibana updates Elasticsearch assets automatically.
This will allow you to safely grant access to more users within your organization.
The required Kibana privileges are `All` for the Fleet and Integrations feature.
```

Since many Integrations assets are shared across spaces, users need the Kibana privileges in all spaces.
Also, we recommend giving users read-only access to Elastic Agent logs and metrics so they can
monitor their agents and troubleshoot problems. We plan to introduce finer-grained controls in later releases.

[role="screenshot"]
image::images/81-privs.png[Fleet now works with limited privileges]

[discrete]
== Fleet offers a cleaner startup experience

Previously, if a user was not using Fleet, we installed several integrations by default.

> We installed many dashboards and visualizations which showed data and cluttered the user experience.

Author: ```suggestion
This over-complicated the setup experience by adding dashboards and visualizations that weren't necessarily useful for all users. 
```

> Even if the user had no agents installed, it also added default agent policies.

Reviewer: ```suggestion
Even if the user had no agents installed, Fleet added default agent policies.
```
